doc/source/whatsnew/v1.2.0.rst

```diff
@@ -354,7 +356,7 @@ Reshaping
 - Bug in :func:`union_indexes` where input index names are not preserved in some cases. Affects :func:`concat` and :class:`DataFrame` constructor (:issue:`13475`)
 - Bug in func :meth:`crosstab` when using multiple columns with ``margins=True`` and ``normalize=True`` (:issue:`35144`)
 - Bug in :meth:`DataFrame.agg` with ``func={'name':<FUNC>}`` incorrectly raising ``TypeError`` when ``DataFrame.columns==['Name']`` (:issue:`36212`)
--
+- Bug in :meth:`Series.transform` would give incorrect results or raise when the argument ``func`` was dictionary (:issue:`35811`)
```

Comment:
`transform` doesn't reshape your series

Reply:
i've co-located it with DataFrame.agg. The issue is about a passing a dictionary of functions to Series.transform, which creates a DataFrame, even if the dict has only one entry. is that not reshaping?

in pandas\doc\source\whatsnew\v0.23.0.rst it was placed in Groupby section. move it there or create a new section?

Reply:
or maybe change section header to Reshaping/reductions/transformations